AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -217,6 +220,21 @@ - (void)fetchData
       _player = [[AVPlayer alloc] initWithPlayerItem:_currentItem];
       _player.muted = _muted;
     }
+    __weak __typeof(self) weakSelf = self;
+    _timeObserver = [_player addPeriodicTimeObserverForInterval:CMTimeMake(1, 1) queue:NULL usingBlock:^(CMTime time){
```

Comment:
An alternative approach is to add a delegate method that informs when the player changes. That would allow the receiver can configure any observers as they desire. `addPeriodicTimeObserverForInterval:`, `addBoundaryTimeObserverForTimes:`, etc